programs/vote/src/vote_state/mod.rs

```diff
@@ -1365,6 +1396,192 @@ mod tests {
         assert_eq!(vote_state.votes.len(), 2);
     }
 
+    #[test]
+    fn test_update_commission() {
+        let node_pubkey = solana_sdk::pubkey::new_rand();
+        let withdrawer_pubkey = solana_sdk::pubkey::new_rand();
```

Comment:
nit: I know that the rest of the tests use `new_rand` everywhere, but these don't need to be random, and can safely use `Pubkey::new_unique()`
```suggestion
        let node_pubkey = Pubkey::new_unique();
        let withdrawer_pubkey = Pubkey::new_unique();
```